# Downloading data

Downloaded "Document 3. Full results of screen against cancer, apoptosis, gene expression, and kinase" xslx file on Februrary 23, 2021 from <https://ash.silverchair-cdn.com/ash/content_public/journal/blood/129/22/10.1182_blood-2017-02-766204/4/blood766204-sup-shrna.xlsx?Expires=1616523511&Signature=bwD63Un4BFvHfczfOvRq8jk80SY4fx9QfRvdlv7CfiaUCxSvjIhLKwTHMYsy~XSVEPzyevAcUa6j1r~tJiTr4q~n6~Ez5RAq7Y2NojDfc8Yu5fh3m2CJi7N0~5zWneQAAZALAWp3a3UL16aYrynvpXfwzfytgW5wEyzDQuIfCLr~cyfkpgUIF3WxNxPkuTOFOV4~oUOno~AmE1pTM~hfG9JuOgfKwQVcGCBE4xYxE1U9B5Y861eNYiyhJ3MNd1qrTScAsmm5fp~v3KX36MMR87b8t0RWMq9Jf27y9GbQA0MpT8RdYgkUw26p1MVuJg46dz9~NhAd2WZSN0OMnbw6hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA>

This direct link to the data appears to no longer be working, but the full electronic article can be viewed [on the Blood website](<https://ashpublications.org/blood/article/129/22/3000/36066/Suppression-of-B-cell-development-genes-is-key-to?searchresult=1>)

# SHA1 for cagek data

abab5a9ad39dc943c06951a21b5df5eb34a51752

# Description of data
This file contains 8 columns or variables in the rho hits sheet, which will be the focus of my project. The first three are related to the gene identification, with Entrez Gene ID, official gene symbol, and gene information respectively. The fourth and fifth columns provide information about the number of shRNAs used for the gene in the screen followed by the number of shRNAs which had sufficient reads following dexamethasone treatment. A large discrepancy between these two columns (i.e. fewer shRNAs with sufficient reads than total shRNAs used in the screen) suggests that knock down of that particular gene is detrimental to overall cell survival (which can be confirmed by observing the gamma phenotypes also available in the xslx file). The seventh column reflects the phenotypic effect of the gene knock down on dexamethasone sensitivity, with more negative numbers indicating greater sensitivity and more positive numbers indicating greater resistance to dexamethasone, and the last column is the p-value for this phenotype. Both the phenotype and p-value were calculated using python scripts provided by the original developers of the screen, whose original manuscript/protocol can be viewed [here](<https://pubmed.ncbi.nlm.nih.gov/24992097/>). These scripts will calculate p-values based on both Mann-Whitney U test and Kolmogorov-Smirnov test, but the Mann-Whitney p-values were used in this figure. 
